components/Core/Paragraph/index.tsx

```diff
@@ -0,0 +1,13 @@
+import ReactMarkdown from 'react-markdown/with-html'
+
+interface ParagraphProps {
+  text: string
+}
+
+export default function Paragraph({ text }: ParagraphProps) {
```

Comment:
use ``const Paragraph:FC<> = (...)`` instead of typecasting into function declaration